test/func_test/test_ctrt/test_atomic_ctrt.py

```diff
@@ -0,0 +1,347 @@
+        """
+        tc = new_taker_tok_ctrt
+        await tc.issue(acnt1, 100)
+        await cft.wait_for_block()
+        return tc
+
+    @pytest.fixture
+    async def new_maker_atomic_swap_ctrt(
+        self,
+        new_maker_tok_ctrt_with_tok: pv.TokenCtrtWithoutSplit,
+        acnt0: pv.Account,
+    ) -> pv.AtomicSwapCtrt:
+        """
+        new_maker_atomic_swap_ctrt is the fixture that registers a new atomic swap contract.
+
+        Args:
+            new_maker_ctrt_with_tok (pv.TokenCtrtWithoutSplit): The fixture that registers a new atomic swap contract and issues tokens right after it.
+            acnt0 (pv.Account): The account of nonce 0.
+
+        Returns:
+            pv.AtomicSwapCtrt: The AtomicSwapCtrt instance.
+        """
+        tc = new_maker_tok_ctrt_with_tok
+
+        tok_id = pv.Ctrt.get_tok_id(tc.ctrt_id, 0)
+        ac = await pv.AtomicSwapCtrt.register(acnt0, tok_id)
+        await cft.wait_for_block()
+
+        await tc.deposit(acnt0, ac.ctrt_id, 100)
+        await cft.wait_for_block()
+
+        return ac
+
+    @pytest.fixture
+    async def new_taker_atomic_swap_ctrt(
+        self,
+        new_taker_tok_ctrt_with_tok: pv.TokenCtrtWithoutSplit,
+        acnt1: pv.Account,
+    ) -> pv.AtomicSwapCtrt:
+        """
+        new_taker_atomic_swap_ctrt is the fixture that registers a new atomic swap contract.
+
+        Args:
+            new_taker_ctrt_with_tok (pv.TokenCtrtWithoutSplit): The fixture that registers a new token contract and issues tokens right after it.
+            acnt1 (pv.Account): The account of nonce 1.
+
+        Returns:
+            pv.AtomicSwapCtrt: The AtomicSwapCtrt instance.
+        """
+        tc = new_taker_tok_ctrt_with_tok
+
+        tok_id = pv.Ctrt.get_tok_id(tc.ctrt_id, 0)
+        ac = await pv.AtomicSwapCtrt.register(acnt1, tok_id)
+        await cft.wait_for_block()
+
+        await tc.deposit(acnt1, ac.ctrt_id, 100)
+        await cft.wait_for_block()
+
+        return ac
+
+    async def test_register(
+        self, acnt0: pv.Account, new_maker_tok_ctrt_with_tok: pv.TokenCtrtWithoutSplit
+    ) -> pv.AtomicSwapCtrt:
+        """
+        test_register tests the method register.
+
+        Args:
+            acnt0 (pv.Account): The account of nonce 0.
+
+        Returns:
+            pv.AtomicSwapCtrt: the AtomicSwapCtrt instance.
+        """
+        tc = new_maker_tok_ctrt_with_tok
+        tok_id = pv.Ctrt.get_tok_id(tc.ctrt_id, 0)
+        ac = await pv.AtomicSwapCtrt.register(acnt0, tok_id)
+        await cft.wait_for_block()
+
+        assert (await ac.maker) == acnt0.addr.b58_str
+        assert (await ac.token_id) == tok_id
+
+        return ac
+
+    async def test_maker_lock_and_taker_lock(
+        self,
+        acnt0: pv.Account,
+        acnt1: pv.Account,
+        new_maker_atomic_swap_ctrt: pv.AtomicSwapCtrt,
+        new_taker_atomic_swap_ctrt: pv.AtomicSwapCtrt,
+    ):
```

Comment:
```suggestion
    ) -> None:
```
The same for other testing methods that don't have a return value.